aws_lambda_powertools/utilities/data_classes/secrets_manager_event.py

```diff
@@ -0,0 +1,20 @@
+from typing_extensions import Literal
+
+from aws_lambda_powertools.utilities.data_classes.common import DictWrapper
+
+
+class SecretsManagerEvent(DictWrapper):
+    @property
+    def secret_id(self) -> str:
+        """SecretId: The secret ARN or identifier"""
+        return self["SecretId"]
+
+    @property
+    def client_request_token(self) -> str:
+        """ClientRequestToken: The ClientRequestToken of the secret version"""
+        return self["ClientRequestToken"]
```

Comment:
```suggestion
    @property
    def client_request_token(self) -> str:
        """ClientRequestToken: The ClientRequestToken of the secret version"""
        return self["ClientRequestToken"]
        
    @property
    def version_id(self) -> str:
        """Alias to ClientRequestToken to get token associated to version"""
        return self["ClientRequestToken"]        
```

the UX is quite confusing due to API interchangeably using `ClientRequestToken` and `VersionId` with the same value in different API calls. We can make it easier for customers (cognitive load) by having an alias here to what the intent actually is.